packages/trader/src/Assets/Trading/Categories/turbos-trade-description.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { localize } from '@deriv/translations';
+import { Text } from '@deriv/components';
+
+export const TurbosTradeDescription = () => {
+    const content = [
```

Comment:
@maryia-deriv Let's move the variable outside of the component and use `as const` 🙏🏻

Reply:
@farzin-deriv unfortunately, translation is not working when the variable is moved outside of this component and currently the type is inferred successfully, so no need for an additional assertion with const.
<img width="464" alt="Screenshot 2023-03-28 at 10 21 35 AM" src="https://user-images.githubusercontent.com/103177211/228159357-01bd7987-24a7-4963-a608-cdc777ab2d98.png">